src/inmanta/protocol/methods.py

```diff
@@ -353,7 +353,7 @@ def upload_file(id: str, content: str):
     """
     Upload a new file
 
-    :param id: The id of the file
+    :param id: The sha1 hash of the content of the file.
```

Comment:
Actually pretty strange that we take this as a parameter, because we also still calculate it (as we should; I was scared for a bit that we'd take the user's word for granted but luckily we don't).